lib/blocking-io.js

```diff
@@ -0,0 +1,6 @@
+module.exports = function () {
+  if (process.stdout._handle && typeof process.stdout._handle.setBlocking === 'function') {
+    process.stdout._handle.setBlocking(true)
+    process.stderr._handle.setBlocking(true)
+  }
```

Comment:
@Fishrock123 trying to go make sure yargs supports all the way back to `0.8` currently:

```js
Benjamins-MBP:yargs benjamincoe$ node --version
v0.8.28
Benjamins-MBP:yargs benjamincoe$ node
> process.stdout._handle.setBlocking
undefined
```

was hoping it would work retroactively, and tried this initially, it does not appear to. You need to have set blocking to true before you begin writing.